README.md

```diff
@@ -1,24 +1,270 @@
-# README
+# Final group capstone - Book an Appointment For Renting Car
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+<div align="center">
+<h2>ER Diagrma</h2>
+<img width="741" alt="Screenshot 2023-09-25 at 12 07 22" src="./ER Diagram Final capstone.PNG">
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents 
 
-* Configuration
+- [Final group capstone - Book an Appointment For Renting Car](#final-group-capstone---book-an-appointment-for-renting-car)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Final group capstone - Book an Appointment ](#-final-group-capstone---book-an-appointment-)
+  - [🚀 Frontend App ](#-frontend-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Run rails](#run-rails)
+  - [👥 Authors ](#-authors-)
+  - [📆 Kanban Board ](#-kanban-board-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+      - [Development Team](#development-team)
+      - [Institution](#institution)
+      - [Reference Design](#reference-design)
+  - [📝 License ](#-license-)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 Final group capstone - Book an Appointment <a name="about-project"></a>
 
-* How to run the test suite
+**[Description]**
 
-* Services (job queues, cache servers, search engines, etc.)
+The project we built for the Final Project is based on an app to book an appointment for the best car in the world. 
 
-* Deployment instructions
+## 🚀 Frontend App <a name="frontend"></a>
+- [Click here to navigate to the Frontend ]()
 
-* ...
```

Comment:
I think you should add the link pointing to the Frontend Project, What do you think?